actions/code-release/action.yml

```diff
@@ -5,47 +5,32 @@ inputs:
    token:
      description: "Token needed to access files "
      required: true
-   path-to-json:
-     description: "Path to json folder with version"
+   access-key:
+     description: "AWS access key"
      required: true
-   repository-name:
-     description: "Name of repository needed to create code packages"
+   secret-key:
+     description: "AWS secret key"
      required: true
-   repository-type:
-     description: "Repository type used in bash code"
+   json-path:
```

Comment:
This is not json-path but a path to a file containing version in json format .. if you call the bellow version-file then this can be version-file-json .. or what ever else but the name is not really clear

Reply:
name changed